lib/velum/salt.rb

```diff
@@ -41,6 +41,26 @@ def self.minions
       JSON.parse(res.body)["return"].first
     end
 
+    # Returns the minions that have not been accepted into the cluster
+    def self.pending_minions
+      res = perform_request(endpoint: "/", method: "post",
+                            data: { client: "wheel",
+                                    fun:    "key.list" ,
+                                    match:  "all",
+                                    tgt:    "ca" })
```

Comment:
I think you can remove tgt from this call, since the wheel client is executed on the master, and this argument will probably be ignored.